tools/wpt/browser.py

```diff
@@ -529,13 +529,12 @@ class ChromeChromiumBase(Browser):
         "Darwin": "Mac",
     }.get(uname[0])
 
-    def _get_latest_chromium_revision(self, architecture):
-        """Queries Chromium Snapshots and returns the latest Chromium revision number
-        for the current platform.
-        """
-        revision_url = ("https://storage.googleapis.com/chromium-browser-snapshots/"
-                        f"{architecture}/LAST_CHANGE")
-        return get(revision_url).text.strip()
+    def _get_latest_chromium_revision(self):
```

Comment:
I would expect this to return an int, but it returns a string. Type annotation to make that very clear:

```suggestion
    def _get_pinned_chromium_revision(self) -> str:
```

You could also parse it to an int, but a string is OK.

Also suggest avoiding "latest", since it may no longer be.